demo/demo.html

```diff
@@ -20,22 +20,14 @@
       const searchDriverPlugin = new sfxPlugins.SearchDriverPlugin();
       // register all plugins with core
       core.register([saytPlugin, saytDriverPlugin, domEventsPlugin, searchPlugin, searchDriverPlugin]);
-
       // search integration/functionality test
-      window.addEventListener("sfx::search_response", (e) => {
+      window.addEventListener('sfx::search_response', (e) => {
```

Comment:
reverted demo.html to old code for Joeys demo branch. Minus the unneeded autocomplete call and the one quotes change